scripts/lib/CIME/case.py

```diff
@@ -65,7 +63,12 @@ class Case(object):
     the case object creates and manipulates the Case env classes
     by reading and interpreting the CIME config classes.
 
+    This class extends across multiple files, class members external to this file
+    are listed in the following imports
     """
+    from CIME.case_setup import case_setup
```

Comment:
By moving these imports into the class and defining them with self instead of case we incorporate them into the class.